### ISSUE TITLE
NSFW porntube.com - Trackers using behind-the-scene whitelisting ?

### ISSUE THREAD

**Issue by YetAnotherGuestAccount** [uBlockOrigin/uAssets#65]
Open uBO logger to display behind-the-scene requests, open on this NSFW site for instance the URL

`http://www.porntube.com/videos/rachel-starr-fucking-husbands-hot-suited-boss_7204765`

then close the tab, then just after closing in the logger an unblocked behind-the-scene xhr request appears to an URL looking like

`http://northstar.porntube.com/tracker.php?uuid=1234567&time=0&user=anonymous&site=porntube`

(parameters values may vary).

In fact before one closes the tab, there is a similar request originating from the tab and blocked by Fanboy's ultimate list's filter

/tracker.php?

but this filter can't block the similar request when it originates from behind-the-scene because behind-the-scene is whitelisted by default by uBO. Now I am not sure that this is a tracker or another nuisance and I am not a very advanced user but I suspect it is and more importantly, that it may be using some not yet widely known trick to avoid blockers which whitelist behind-the-scene requests. If I'm not totally wrong maybe the trick could be looked at and the behind-the-scene default whitelisting could be reconsidered ? I realize that to handle this case an advanced user can remove the behind-the-scene whitelisting and replace the default dynamic filtering rules with something like

behind-the-scene \* \* allow
behind-the-scene porntube.com \* noop

to let the static filters block what needs to be, but what about other users and sites if that sort of trick becomes common ? I am not sure what could be done to solve the problem but my purpose here is only to report it.
### Versions
- Browser/version: Firefox 47.0
- uBlock Origin version: 1.7.0
### Settings
- I added a bunch of static filter lists from the proposed ones or made by myself, I won't give them all unless it becomes relevant to know

**Comment by gorhill** [comment]
This is a code issue, not a filter issue.

The XHR is fired when the page at `beforeunload` time, so the tab has been discarded at this point and can't be found when the HTTP observer has to handle the network request. Do you mind opening an issue at <https://github.com/gorhill/uBlock/issues> and copying the one here at this new issue?

**Comment by rusingineer** [comment]
add to main filter:
||northstar.porntube.com/tracker.php
||porntube.com/adc/porntube_b2.php
||porntube.com/adc/pop_ex.php
